katalog/istio/README.md

```diff
@@ -17,169 +34,175 @@ Source: [https://istio.io/docs](https://istio.io/docs/concepts/what-is-istio/#wh
 ```
 
-Change the `external_services.prometheus.url` value with the correct prometheus server url.
+##### Note
+
+This package will deploy telemetry + service monitors. If telemetry it's already deployed, replace the telemetry base
+with the `katalog/istio/telemetry/service-monitor` base.
+
+
+### Egress Gateway
+
+This package deploys on top of the [minimal installation](#minimal-installation),
+[the egress gateway component](egress-gateway/README.md).
+
+Once deployed this package, don't forget to deploy the new Istio configuration based on the components installed:
+
+[Example kustomize installation file](../../examples/istio/minimal-and-egress/kustomization.yaml):
+
+```yaml
+namespace: istio-system
+
+bases:
+  - katalog/istio/init
+  - katalog/istio/minimal
+  - katalog/istio/egress-gateway
+  - katalog/istio/sidecar-injection/configuration/minimal-and-egress
+```
+
+### Citadel
+
+[Citadel](citadel/README.md) is the Istio component in charge of provide security features *(key and certificate management)*.
+If you are looking for mTLS, this is the component you should install.
+
+As the other components, this one should be installed on top of the minimal istio deployment.
+
+Example kustomize installation file:
+
+```yaml
+namespace: istio-system
+
+bases:
+  - katalog/istio/init
+  - katalog/istio/minimal
+  - katalog/istio/citadel
+  - katalog/istio/egress-gateway
+  - katalog/istio/sidecar-injection/configuration/citadel-and-egress
+```
+
+In this example you can see istio + citadel + egress gateway installed.
+
+##### Note
+
+Be sure to deploy the correct `sidecar-injection/configuration` kustomize project. In this example the correct
+configuration is the one with [citadel + egress](sidecar-injection/configuration/citadel-and-egress).
+
+
+### Sidecar injector
+
+[Sidecar injector](sidecar-injection/README.md) is the component responsible of injecting the envoy proxy on every
+required pod. This is the alternative of executing `istioctl inject` comamnds.
+
+Sidecar injector package requires [citadel](citadel/README.md) installed.
+
+Example kustomize installation file:
+
+```yaml
+namespace: istio-system
+
+bases:
+  - katalog/istio/init
+  - katalog/istio/minimal
+  - katalog/istio/citadel
+  - katalog/istio/egress-gateway
+  - katalog/istio/sidecar-injection
+  - katalog/istio/sidecar-injection/configuration/sidecar-injection-and-egress
+```
+
+In this example you can see istio + citadel + egress gateway and the sidecar injector installed.
+
+##### Note
+
+Be sure to deploy the correct `sidecar-injection/configuration` kustomize project. In this example the correct
+configuration is the one with [sidecar-injector + egress](sidecar-injection/configuration/sidecar-injection-and-egress).
+
+### Kiali
+
+[Kiali](kiali/) is a project used to visualize the service mesh in a graphical way. It needs metrics stored in a prometheus
+server. So it requires to have installed telemetry package + optionally the service-monitor subpackage.
```

Comment:
- project or module? maybe both in this case?
- From the project website I see that is a little more than "a GUI for the service mesh" and that also explains why it needs prometheus? Maybe refrase to (I'm just copy&pasting from the project website with a little editing):

[Kiali](kiali/) is a project that implements an observability console for Istio. It helps you to understand the structure of your service mesh by inferring the topology, the health of your mesh and see detailed metrics. Distributed tracing is provided by integrating Jaeger.

In order to run Kiali the telemetry package (module?) must be installed previously and optionally the service-monitor subpackage to enable <INSERT HERE WHAT DOES INSTALLING THE OPTIONAL PACKAGE ENABLE>.

Reply:
In this case, kiali is another project as it is not part of Istio. So, it is a project and a package. 

Copied you Kialia description ;)